From cac1fffce80f0835e4d8d234023a775b4243b916 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20Br=C3=BCns?= <stefan.bruens@rwth-aachen.de>
Date: Sun, 20 Oct 2019 01:44:02 +0200
Subject: [PATCH] Create correct pkconfig file for zbar-qt5

Although zbar supports Qt5, the created pkgconfig file always requested
the Qt(4) components.

Fixes issue #62.
---
 configure.ac   | 13 ++++++++++---
 zbar-qt5.pc.in | 12 ++++++++++++
 2 files changed, 22 insertions(+), 3 deletions(-)
 create mode 100644 zbar-qt5.pc.in

diff --git a/zbar-qt5.pc.in b/zbar-qt5.pc.in
new file mode 100644
index 0000000..3378993
--- /dev/null
+++ b/zbar-qt5.pc.in
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: zbar-qt
+Description: bar code scanning and decoding Qt5 widget
+URL: http://zbar.sourceforge.net
+Version: @VERSION@
+Requires: zbar, Qt5Core >= 5, Qt5Gui >= 5
+Libs: -L${libdir} -lzbarqt
+Cflags: -I${includedir}
-- 
2.23.0

